### PR TITLE
Apr 2138 oversight gateway fail state change

### PR DESCRIPTION
**Pull request by chrisfoster76**

### CODE REVIEW

File: src/SFA.DAS.RoatpOversight.Web/Exceptions/OversightReviewStatusExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,54 @@
+﻿using SFA.DAS.RoatpOversight.Domain;
```
Comment (MarkFCain) [inline src/SFA.DAS.RoatpOversight.Web/Exceptions/OversightReviewStatusExtensions.cs:1]: should this be in an 'extensions' folder rather than exceptions?

**Review by MarkFCain — APPROVED** [review]
just a comment about the extension location

File: src/SFA.DAS.RoatpOversight.Web.UnitTests/Services/ApplicationOutcomeOrchestratorTests.cs [hunk 2]
```diff
@@ -57,6 +57,11 @@ public void Before_each_test()
 
             _applicationApiClient.Setup(x => x.RecordOutcome(It.Is<RecordOversightOutcomeCommand>(y => y.ApplicationId == _applicationId))).ReturnsAsync(true);
 
+            _applicationApiClient
+                .Setup(x => x.RecordGatewayFailOutcome(
+                    It.Is<RecordOversightGatewayFailOutcomeCommand>(y => y.ApplicationId == _applicationId)))
+                .Returns(Task.CompletedTask);
+
```
Comment (martin-lush) [inline src/SFA.DAS.RoatpOversight.Web.UnitTests/Services/ApplicationOutcomeOrchestratorTests.cs:64]: This feels a bit dubious as I suspect this is an Async method but the setup has been done as if it's a synchronous one

Comment (chrisfoster76) [reply src/SFA.DAS.RoatpOversight.Web.UnitTests/Services/ApplicationOutcomeOrchestratorTests.cs:61]: The setup is to return a Task.

**Pull request merged**